src/app/core/services/notification.service.ts

```diff
@@ -15,8 +15,8 @@ export class Notification {
     message: string | TemplateRef<any>,
   ) {
     this.message = message;
-    this.autohide = false;
-    this.delay = 0;
+    this.autohide = true;
```

Comment:
Either use the setAutohide https://github.com/JamFactoryApp/jamfactory-web/blob/e56023d8b37830b67ddf9221c32051b896679f35/src/app/core/services/notification.service.ts#L37 function to accomplish the autohide behavior or invert the logic everywhere and add a removeAutohide function for notifications that should stay until the user closes them.

Reply:
Is there a scenario where the notification should always be shown and not disappear after a given time?

Reply:
- Notification that the JamSession was closed due to inactivity and
- Notification to indicate that the user should select a playback device 
are two examples that are already implemented

You can also use
https://github.com/JamFactoryApp/jamfactory-web/blob/e56023d8b37830b67ddf9221c32051b896679f35/src/app/core/services/notification.service.ts#L72 to clear all notifications, https://github.com/JamFactoryApp/jamfactory-web/blob/e56023d8b37830b67ddf9221c32051b896679f35/src/app/core/services/notification.service.ts#L80 to clear all notifications without autohide or https://github.com/JamFactoryApp/jamfactory-web/blob/e56023d8b37830b67ddf9221c32051b896679f35/src/app/core/services/notification.service.ts#L76 to delete a specific notification

Reply:
I think it's better to increase the duration for which these notifications are visible and not make them stay there forever.

Another solution, as a compromise, would be to implement a notification box that can be shown when clicking a button and which contains unseen notifications as well as a badge indicating how many notifications have not been seen by the user yet.

On the other hand this seems like overkill in contrast to the amount of notifications that we will show to the user.

Reply:
I've opend a new issue. @Pallemann please remove the changes to the notification behaviour and resume the discussion in the issue.